Makefile.sp

```diff
@@ -114,7 +115,24 @@ sp-vale: sp-install
 	@echo ""
 	@. $(VENV); vale --config "$(SPHINXDIR)/vale.ini" --glob='*.{md,txt,rst}' $(TARGET)
 
-
+sp-pdf-prep: sp-install
+	@for packageName in $(REQPDFPACKS); do (dpkg-query -W -f='$${Status}' $$packageName 2>/dev/null | \
+        grep -c "ok installed" >/dev/null && echo "Package $$packageName is installed") || \
+        (echo "PDF generation requires the installation of the following packages: $(REQPDFPACKS)" && \
+        echo "" && echo "sudo make pdf-prep-force will install these packages" && echo "" && echo \
```

Comment:
I think I misread this in the previous version and thought it instructed you how to install the one missing package - which would make more sense in this loop. Right now you just get the same message all over again.

Could we change it to give an instruction for the respective package (and also mention pdf-prep-force) - something like "Run sudo apt-get install $(REQPDFPACKS) to install it, or sudo make pdf-prep-force to install all missing packages"?

Reply:
I will break the loop if packages are not found - it's probably the easiest way. 

And good point, I'll change the text.

Reply:
Fixed with [9e0a32e](https://github.com/canonical/sphinx-docs-starter-pack/pull/264/commits/9e0a32ecedc9022d4174d8d57a3308820b5b8f2e).